easybuild/easyblocks/a/abaqus.py

```diff
@@ -102,12 +121,17 @@ def install_step(self):
                 r"Default.*SIMULIA/Commands\]:\s*": os.path.join(self.installdir, 'Commands'),
                 r"Default.*SIMULIA/CAE/plugins.*:\s*": os.path.join(self.installdir, 'plugins'),
                 r"Default.*SIMULIA/Isight.*:\s*": os.path.join(self.installdir, 'Isight'),
-                r"License Server [0-9]+\s*(\n.*){3}:": 'abaqusfea',  # bypass value for license server
-                r"License Server . \(redundant\)\s*(\n.*){3}:": '',
-                r"Please choose an action:": '1',
                 r"SIMULIA/Tosca.*:": os.path.join(self.installdir, 'tosca'),
                 r"location of your existing ANSA installation.*(\n.*){8}:": '',
                 r"FLUENT Path.*(\n.*){7}:": '',
+
+                # License server
+                r"License Server [0-9]+\s*(\n.*){3}:": 'abaqusfea',  # bypass value for license server
+                r"License Server . \(redundant\)\s*(\n.*){3}:": '',
+                r"Please choose an action:": '1',
+
+                # Continue
+                nextstr: '',
```

Comment:
@branfosj One thing to be aware of here is that there's no order over these patterns (because it's a `dict` value).

So if the pattern in `nextstr` is matched, you won't match any other more specific patterns that also include the `nextstr` pattern...